includes/admin/class-wp-job-manager-writepanels.php

```diff
@@ -83,6 +83,7 @@ public function job_listing_fields() {
 			$fields['_job_expires'] = array(
 				'label'       => __( 'Listing Expiry Date', 'wp-job-manager' ),
 				'priority'    => 11,
+				'classes'     => ['wpjm-date'],
```

Comment:
May not be good to be using shorthand array syntax as that is only supported on PHP 5.4+